tests/acceptance/features/bootstrap/AuthContext.php

```diff
@@ -607,7 +607,7 @@ public function userRequestsEndpointsUsingTheGeneratedAppPassword($method, Table
 	 * @return void
 	 */
 	public function sendRequest(
-		$url, $method, $authHeader = null, $useCookies = false, $body = null
+		$url, $method, $authHeader = null, $useCookies = false, $body = null, $headers = null
```

Comment:
```suggestion
		$url, $method, $authHeader = null, $useCookies = false, $body = null, $headers = []
```

Should `$headers` default to an empty array? (like in the deleted line below)